examples/gp.py

```diff
@@ -0,0 +1,131 @@
+import matplotlib
+matplotlib.use('Agg')  # noqa: E402
+import matplotlib.pyplot as plt
+import jax
+
+import argparse
+
+import numpy as onp
+from jax import vmap
+import jax.numpy as np
+import jax.random as random
+
+import numpyro.distributions as dist
+from numpyro.handlers import sample
+from numpyro.hmc_util import initialize_model
+from numpyro.mcmc import mcmc
+
+from jax.config import config
+config.update('jax_enable_x64', True)
```

Comment:
Does this at all work with float32 precision?

Reply:
Good question, I am curious too. Looking at plot of the data, I think it will work.

Reply:
this seems sort of necessary i was getting some weird results without

Reply:
I'll look closer to see if there is any related issue.